adjust platform description in README (#605)
wormhole-foundation/wormhole-sdk-ts: README.md

## Usage

> Getting started is simple, just import the 'meta' Wormhole package, that makes sure all [Platform](#platforms) modules are installed.

Reviewer: ```suggestion
Getting started is simple, just import the `@wormhole-foundation/sdk` package, which makes sure all [Platform](#platforms) modules are installed.
```
